### PR TITLE
Modified Mocha Tests

### PR DESCRIPTION
**Pull request by yzhang175**
Hi, guys. I have modified mocha tests:

- made a test file for utils.js (test connect, getRecord and capitalizeName)
- made a test file for fetch country data (China as example, and fetch all elements such as name, language, etc)

However, the build fails. I have fixed linter errors, and added timeout, but it is still fail. All the tests pass on my terminal.
Screen Shot 2021-05-10 at 2 52 36 PM

### CODE REVIEW

File: test/utils.test.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    return request(app)
+      .get('/')
+      .expect(200)
+  });
+});
+
+//Test getRecordFromDB()
+describe('Test getRecordFromDB()', function() {
+  this.timeout(10000);
+  it('should return a country nane from DB', async() => {
```
Comment (wxie11) [inline test/utils.test.js]: typo; should be "name"

Comment (wxie11) [reply test/utils.test.js]: "should return related data of the requested country"

Comment (yzhang175) [reply test/utils.test.js]: Got it!

**Comment by yzhang175** [comment]
Updates:
As we discussed, I have fixed all the possible things. Disconnection is added. 
It works on my terminal without any timeout issues, it actually run fast.

Screen Shot 2021-05-11 at 1 34 09 AM

HOWEVER, the build still fails. It shows the timeout issue on Travi CI.
Screen Shot 2021-05-11 at 1 33 29 AM

**Review by wxie11 — APPROVED** [review]

**Review by jhuang60 — APPROVED** [review]

**Pull request merged**